OmiseSDK/PaymentChooserViewController.swift

```diff
@@ -70,6 +74,8 @@ extension PaymentChooserOption {
             return [.truemoney]
         case .installmentFirstChoice, .installmentKBank, .installmentKTC, .installmentBBL, .installmentBAY, .installmentSCB:
             return [.installment]
+        case .mobileBankingSCB:
```

Comment:
Do you need to add `.mobileBankingSCB` case into `func staticIndexPath(forValue value: PaymentChooserOption) -> IndexPath` too.

Reply:
Thank you. This is done.